src/main/java/org/terasology/gookeeper/system/EconomySystem.java

```diff
@@ -15,34 +15,52 @@
  */
 package org.terasology.gookeeper.system;
 
+import com.google.common.base.Joiner;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.terasology.assets.ResourceUrn;
 import org.terasology.assets.management.AssetManager;
+import org.terasology.entitySystem.Component;
 import org.terasology.entitySystem.entity.EntityManager;
 import org.terasology.entitySystem.entity.EntityRef;
 import org.terasology.entitySystem.event.ReceiveEvent;
 import org.terasology.entitySystem.prefab.Prefab;
 import org.terasology.entitySystem.prefab.PrefabManager;
-import org.terasology.entitySystem.systems.BaseComponentSystem;
-import org.terasology.entitySystem.systems.RegisterMode;
-import org.terasology.entitySystem.systems.RegisterSystem;
-import org.terasology.entitySystem.systems.UpdateSubscriberSystem;
-import org.terasology.gookeeper.component.EconomyComponent;
-import org.terasology.gookeeper.component.GooeyComponent;
-import org.terasology.gookeeper.component.PenBlockComponent;
-import org.terasology.gookeeper.component.VisitBlockComponent;
+import org.terasology.entitySystem.systems.*;
+import org.terasology.gookeeper.component.*;
```

Comment:
Star imports detected.

These should be avoided everywhere. If intelliJ keeps adding them there is a setting for how many imports before it'll collapse them into a star. Set it to 99 and it'll never trigger

Reply:
exterminate! exterminate! exterminate! :dalek:

Reply:
Agreed. Star imports may make the file look nicer, but please do avoid them everywhere anyway.